cmd/tracee-ebpf/internal/config/config.go

```diff
@@ -0,0 +1,80 @@
+package config
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+)
+
+type TraceeEbpfCliConfig struct {
+	PerfRingBuffers struct {
+		EventSubmitBufferSize int
+		BlobBufferSize        int
+	}
+	VerboseDebug bool
+	InstallPath  string
+	BuildPolicy  string
+	Trace        []string
+	Output       []string
+	Capture      []string
+}
+
+var Config TraceeEbpfCliConfig
+
+func LoadConfig(fileName string) error {
+	cfgFile, err := os.Open(fileName)
+
+	if err != nil {
+		return err
+	}
+
+	defer cfgFile.Close()
+
+	byteValue, err := ioutil.ReadAll(cfgFile)
+
+	if err != nil {
+		return err
+	}
+
+	err = json.Unmarshal(byteValue, &Config)
+
+	return err
+}
+
+func DefaultConfigLocation() string {
+	envLocation := os.Getenv("TRACEE_EBPF_CONFIG")
+	if envLocation != "" {
+		return envLocation
+	}
+	return "./tracee-ebpf-config.json"
```

Comment:
This doesn't work as intended: the `.` refers to the working dir of the executing user instead of the location of the executable itself

Reply:
irrelevant now